### PR TITLE
#643 fix typo for assistant, rework delivery assistant page

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
This PR adds docs for delivery assistant for 0.7.x (and 0.8.x, though that will be WIP anyway). For this I have moved the delivery assistant page from reference/bridge to continuous-delivery, added a section about CLI and API.

In addition, I fixed the typo "assistent", and I removed the `"id"` field of the cloud-event when sending a start-evaluation cloud-event via the API.

Preview: https://deploy-preview-658--keptn.netlify.app/docs/0.7.x/continuous_delivery/delivery_assistant/

### CODE REVIEW

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: Delivery Assistant
+description: Approval of deployment for manual approval strategy
+weight: 50
+keywords: [0.7.x-bridge, 0.7.x-cd]
+aliases:
+  - /docs/0.7.x/reference/bridge/delivery_assistent/
+---
+
+If you configured approval-strategy `manual` within your [multi-stage delivery](../multi_stage/#approval-strategy), Keptn's `gateekeper-service` will ask for an approval before deploying the artefact in the current stage, e.g.:
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:10]: I would delete the name `gatekeeper-service` --> to: ", Keptn will ask ..."

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+{{< popup_image
+  link="./assets/bridge_approval_triggered.png"
+  caption="Keptn Bridge Approval Triggered"
+  width="764px">}}
+
+
+You can approve or decline this deployment by sending an `approval.finished` CloudEvent via the Keptn CLI, Keptn Bridge or directly via the API.
+
+## CLI
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:21]: Change heading to: ## Approve or decline an open approval with Keptn CLI

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    1
+    Do you want to (a)pprove or (d)ecline: 
+    a
+    ID of Keptn context: 654c80b0-4a02-4d36-96f3-7447df1cdf41
+    ```
+
+
+## Bridge
+ 
+* To confirm/decline an approval, open the *Environment* view, click on the stage that has a pending approval. When there are pending approvals for a service available, the service tile is expanded automatically. Additionally you can filter the services, to see only services with pending approvals, by clicking on the blue icon.
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:67]: We should keep the wording aligned: To approve/decline an approval ...

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+     (1)	0.11.2	n/a		
+    Select the option to approve or decline: 
+    1
+    Do you want to (a)pprove or (d)ecline: 
+    a
+    ID of Keptn context: 654c80b0-4a02-4d36-96f3-7447df1cdf41
+    ```
+
+
+## Bridge
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:65]: Change heading to: ## Approve or decline an open approval with Keptn Bridge

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md (hunk 3 above)
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:67]: Please exchange with: 

To approve/decline an approval, open the *Environment* view and click on the stage that has a pending approval. When there are pending approvals for a service available, the service tile is expanded automatically. Additionally, you can filter the services to see only services with pending approvals by clicking on the blue icon.

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+ 
+* To confirm/decline an approval, open the *Environment* view, click on the stage that has a pending approval. When there are pending approvals for a service available, the service tile is expanded automatically. Additionally you can filter the services, to see only services with pending approvals, by clicking on the blue icon.
+
+* By clicking on the green checkmark, the deployment will be approved. You can also decline by clicking on the red abort button. 
+
+    {{< popup_image
+      link="./assets/bridge_environment_assisteddelivery.png"
+      caption="Keptn Bridge Assisted Delivery">}}
+
+## API
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:75]: Change heading to: ## Approve or decline an open approval with Keptn API

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    "project": "ck-sockshop",
+    "service": "carts",
+    "stage": "production",
+    "tag": "0.11.2",
+    "teststrategy": "performance"
+  }
+}
+```
+
+* Trigger the approval finished event via request on `/event`:
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:111]: I would not say "Trigger", but rather: "Send the approval finished event to the `/event` endpoint.

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+```console
+curl -X POST "${KEPTN_ENDPOINT}/v1/event" \
+-H "accept: application/json; charset=utf-8" \
+-H "x-token: ${KEPTN_API_TOKEN}" \
+-H "Content-Type: application/json; charset=utf-8" \
+-d @./approval_finished.json
+```
+
+* The endpoint returns a unique ID (`keptn-context`) that is required to retrieve the evaluation result. (*Note:* The response also contains a *token* that is required to open a WebSocket communication. This token is not needed now.)
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:121]: This sentence is obsolete.

**Review by johannes-b — COMMENTED** [review]
I left a few comments on the file: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md

Can you please consider the changes also in: `content/docs/0.8.x`

**Comment by christian-kreuzberger-dtx** [comment]
Thanks, I've made the changes as described in the review.

File: content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+     (1)	0.11.2	n/a		
+    Select the option to approve or decline: 
+    1
+    Do you want to (a)pprove or (d)ecline: 
+    a
+    ID of Keptn context: 654c80b0-4a02-4d36-96f3-7447df1cdf41
+    ```
+
+
+## ## Approve or decline an open approval with Keptn Bridge
```
Comment (johannes-b) [inline content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md:65]: Please remove ##

File: content/docs/0.8.x/continuous_delivery/delivery_assistant/index.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+     (1)	0.11.2	n/a		
+    Select the option to approve or decline: 
+    1
+    Do you want to (a)pprove or (d)ecline: 
+    a
+    ID of Keptn context: 654c80b0-4a02-4d36-96f3-7447df1cdf41
+    ```
+
+
+## ## Approve or decline an open approval with Keptn Bridge
```
Comment (johannes-b) [inline content/docs/0.8.x/continuous_delivery/delivery_assistant/index.md:65]: Same as for file in 0.7.x

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**